docs/dsp-tools-xmlupload.md

```diff
@@ -427,76 +428,78 @@ Example of a property with a public and a hidden decimal value:
 ### &lt;geometry-prop&gt;
```

Comment:
I reorganised the docs of <geometry-prop>. In principle I just moved information from one place to the other, with only few real deletions.